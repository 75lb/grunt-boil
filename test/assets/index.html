<html>
<body>
  <ul>
    <% boil.index.options.someFiles.forEach(function(file){ %><li><a href="<%= file %>"><%= file %></a></li>
    <% }) %>
  </ul>
</body>
</html>