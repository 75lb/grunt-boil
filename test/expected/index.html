<html>
<body>
  <ul>
  
    <li><a href="Gruntfile.js">Gruntfile.js</a></li>
  
    <li><a href="index.html">index.html</a></li>
  
  </ul>
</body>
</html>
